src/ssl.c

```diff
@@ -30672,6 +30640,43 @@ int wolfSSL_RSA_sign_ex(int type, const unsigned char* m,
             WOLFSSL_MSG("This NID (md type) not configured or not implemented");
             return 0;
     }
+    return type;
+}
+
+/* return SSL_SUCCESS on ok, 0 otherwise */
+int wolfSSL_RSA_sign(int type, const unsigned char* m,
+                           unsigned int mLen, unsigned char* sigRet,
+                           unsigned int* sigLen, WOLFSSL_RSA* rsa)
+{
+    return wolfSSL_RSA_sign_ex(type, m, mLen, sigRet, sigLen, rsa, 1,
+            RSA_PKCS1_PADDING);
+}
+
+int wolfSSL_RSA_sign_ex(int type, const unsigned char* m,
```

Comment:
Please add documentation either doxy style above or in the "doc/dox_comments/header_files/rsa.h". For example the "int flag" argument is not clear.